### PR TITLE
Functionalities to do list

### PR DESCRIPTION
**Pull request by lisandroseia**
In this branch we made:
Functionality to add tasks;
Functionality to remove tasks;
Functionality to edit existing tasks;
Functionality to display tasks in screen;
Functionality to save everything in local storage and display it;

### CODE REVIEW

File: src/collection.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+populateStorage() {
+  localStorage.setItem(
+    'tasksList',
+    JSON.stringify({
+      tasksList: this.list,
+    }),
+  );
+}
+}
```
Comment (rotimiazeez) [inline src/collection.js:95]: - [ ] Great job implementing the add and edit functionalities 🔥, I can't remove an item(s) from the list yet kindly see to that😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @lisandroseia 👍, 
## Good job so far!🔥
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
### There are some issues that you still need to work on to go to the next project but you are almost there!👌
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/collection.js (hunk 1 above)
Comment (lisandroseia) [reply src/collection.js:95]: Hi, can you elaborate a little bit more? i can do it properly on my computer, so i cant fix it because i cant see the error

Comment (lisandroseia) [reply src/collection.js:95]: sorry, for deleting something you should click on the text and the button will appear, like in the page that we are cloning, I made the container of the text bigger so the user can use it in a more friendly way, sorry for bothering! :smile:

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @lisandroseia,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lisandroseia** [comment]
Thanks for everything, I appreciate the reviews!

**Pull request merged**